src/table/runtime.rs

```diff
@@ -31,14 +63,142 @@ impl RuntimeServices {
             None => (0, ptr::null()),
         };
 
-        (self.reset)(rt, status, size, data)
+        unsafe { (self.reset)(rt, status, size, data) }
     }
 }
 
 impl super::Table for RuntimeServices {
     const SIGNATURE: u64 = 0x5652_4553_544e_5552;
 }
 
+/// The current time information
+#[repr(C)]
```

Comment:
Done. I did not derive (Partial)Eq for Time because I am not sure if an autoderived Eq is a good idea for this type. It has padding bytes that may be repurposed in the future, and deciding what it means to times to be equal across timezones sounds like a possible headache. Moreover, I am not sure how often people will actually need to compare time measurements for equality.